src/content/operations/known-issues/_index.en.md

```diff
@@ -20,3 +20,7 @@ weight = 40
 * Globalnet is not compatible with Headless Services. Only ClusterIP Services are supported at this time.
 * Currently, Globalnet is not supported with the OVN network plug-in.
 * The `subctl benchmark latency` command is not compatible with Globalnet deployments at this time.
+
+## VXLAN
```

Comment:
Also, I don't think that VXLAN needs its our section. I suggest to use the "General" section above